article/_posts/2019-10-17-StraightToThePoint.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: review
+title: "Straight to the point: reinforcement learning for user guidance in ultrasound"
+tags: deep-learning reinforcement cardiac
+author: "Nathan Painchaud"
+cite:
+    authors: "Fausto Milletari, Vighnesh Birodkar, Michal Sofka"
+    title:   "Straight to the point: reinforcement learning for user guidance in ultrasound"
+    venue:   "MICCAI 2019 (SUSI)"
+pdf: "https://arxiv.org/pdf/1903.00586.pdf"
+---
+
+
+# Highlights
+- System to guide inexperienced users towards the acquisition of clinically relevant images of the heart in ultrasound
+- Environments to simulate offline any (reasonable) acquisition trajectory the user may take while scanning the patient
+  by making use of tracked video sequences
+- Comparison of the performances of the method with the ones obtained by training a classifier to learn a policy on the
+  same data in a fully supervised manner
+
+
+# Introduction
+The authors propose a method for guiding inexperienced users in the acquisition through the parasternal long axis (PLAx)
+sonic window on the heart. The degrees of freedom covered by the guidance are detailed in Figure 2.
+
+![](/article/images/StraightToThePoint/figure2.png)
+
+
+# Methods
+
+## Environment
+It would be impossible to collect data according to the RL agent's policy since it would require to scan a patient
+during training, so a large number of spatially tracked video frames was acquired to navigate the chest area offline.
+
+Table 1 shows how the acquisition's degrees of freedom where represented in possible actions, while Table 2 describes
+the reward strategy.
+
+![](/article/images/StraightToThePoint/table1.png)
+
+![](/article/images/StraightToThePoint/table2.png)
+
+## RL Agent
+The agent is trained using the DQN learning paradigm, with the policy network being a CNN. The input of the model are
+ultrasound images, and its output is represented by nine Q-values, one for each action. Figure 4 shows the proposed
+architecture for the policy network.
+
+![](/article/images/StraightToThePoint/figure4.png)
+
+The objective function used to update the parameters is the following equation:
+
+![](/article/images/StraightToThePoint/equation1.png)
+
+## Supervised policy learning
+Each image acquired in each bin was labelled with one action, which would be the optimal action to perform in that
```

Comment:
What bins ?

Reply:
I detailed what bins are, as well as the environment representation and data acquisition process in general, in the "Environment" section.